src/main/java/com/sap/sgs/phosphor/fosstars/model/feature/oss/OssFeatures.java

```diff
@@ -216,4 +219,19 @@ private OssFeatures() {
    */
   public static final Feature<Boolean> SIGNS_ARTIFACTS
       = new BooleanFeature("If a project signs artifacts");
+
+  /**
+   * If OWASP Dependency plugin is used to scan the project. It is either used as a mandatory step,
+   * optional step or not used at all.
+   */
+  public static final Feature<Status> OWASP_DEPENDENCY_CHECK_SCAN_AVAILABILITY =
+      new AvailabilityFeature("OWASP Dependency Check scan mode");
+
+  /**
+   * Check if configuration to fail the build exists in OWASP Dependency. Identify the CVSS
+   * threshold score to fail the build on finding vulnerabilities greater than the threshold.
+   */
+  public static final Feature<Double> OWASP_DEPENDENCY_CHECK_FAIL_CVSS_THRESHOLD =
+      new DoubleFeature(
+          "OWASP Dependency check CVSS Threshold score used to fail the build");
```

Comment:
```suggestion
          "A CVSS threshold for OWASP Dependency Check to fail the build");
```

Reply:
The threshold belongs to the interval [0, 10] and it can be also not set at all. How about defining a special feature for that? Let's set `OwaspDependencyCheckCvssThreshold`.

Reply:
Is it better to have this `BoundedDoubleFeature` alternative to already existing `BoundedIntegerFeature`?

Reply:
I meant something different but I am not sure that I have a clear understanding how it can be implemented. Feel free to add a new `BoundedDoubleFeature` class. It would be also good to define a constant for the `11` score in single place. However, I am not sure where it should be.